src/main/app/src/pages/HelpPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import '../css/home.css';
+
+// Import Bootstrap CSS and JS. Only this page uses these. 
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/js/bootstrap.js';
+
+// This is a stateless, functional React component used to render each resource in a card format. 
+const ResourceCard = ({ organization, description, url }) => (
+  <div className = "p-3">
+    <div className="card">
+      <div className="card-body">
+        <h5 className="card-title text-info">{organization}</h5>
+        <p className="card-text text-secondary" >{description}</p>
+        <a className="card-text btn btn-light" href={url}>View more</a>
+      </div>
+    </div>
+  </div>
+);
+
+/* Component for home page */
+const HelpPage = () => {
+
+  const [resources, updateResources] = React.useState([]);
+
+  useEffect(() => {
```

Comment:
the useEffect hook is called after the component is rendered. Since the call to this api is the content that is to be rendered, I think it would be better to just call it in the component body itself before render. Since whenever state changes, the component re-renders.